api/src/main/kotlin/com/gmtkgamejam/models/Skills.kt

```diff
@@ -0,0 +1,33 @@
+package com.gmtkgamejam.models
+
+import kotlin.math.pow
+
+enum class Skills(val bitwiseId: Int) {
+    `2DArt`(1),
+    `3DArt`(2),
+    Code(4),
+    DesignProduction(8),
+    SoundMusic(16),
+    TestingSupport(32),
+    TeamLead(128),
+    Other(64);
+
+    companion object {
+        private val map = values().associateBy(Skills::bitwiseId)
+        private fun fromInt(type: Int) = map[type]
+
+        fun fromBitwiseId(id: Int): List<Skills> {
+            val skills: ArrayList<Skills> = arrayListOf();
+
+            // Range should be 0..{number of Skills - 1}
+            for (i in 0..7) {
```

Comment:
Why not loop over values() and extract the bitwiseId value from each item in the loop?

Reply:
That is a much cleaner solution, good call - I'll redo this loop to use that, cheers :)